app/Jobs/BroadcastNotification.php

```diff
@@ -186,6 +187,29 @@ private function onBeatmapsetResetNominations()
         ];
     }
 
+    private function onChannelMessage()
+    {
+        $channel = Channel::findOrFail($this->object->channel_id);
+        $users = $channel->users()->pluck('user_id');
+
+        $this->receiverIds =
+            array_values(
+                array_filter(
+                    $users->toArray(),
+                    function ($k) {
+                        return $k !== $this->source->user_id;
```

Comment:
This is already done somewhere else (when creating per-user notification and before sending it to client).